app/src/main/AndroidManifest.xml

```diff
@@ -10,6 +10,13 @@
     <uses-permission android:name="android.permission.FOREGROUND_SERVICE" />
     <uses-permission android:name="android.permission.DOWNLOAD_WITHOUT_NOTIFICATION" />
 
+    <queries>
+        <intent>
+            <action android:name="android.intent.action.VIEW" />
+            <data android:scheme="http" />
```

Comment:
Does that work for `https` links as well?

Reply:
We do use that schema to create a "fakeIntent" for `http://someurl.com` to query browser apps. I will add "https" too just to have the set complete.